code/world.dm

```diff
@@ -18,7 +18,7 @@ var/global/datum/global_init/init = new ()
 	cache_lifespan = 0	//stops player uploaded stuff from being kept in the rsc past the current session
 
 
-#define RECOMMENDED_VERSION 508
+#define RECOMMENDED_VERSION 510 //Goddammit, update this once in a while!

```

Comment:
Shitposting? I want to remind people who see it on accident if it needs updating, that's all.